shells/pipes-shell/source/testDevice.js

```diff
@@ -105,13 +105,21 @@ const smokeTest = async (bus) => {
     send({message: 'spawn', modality: 'dom', recipe: 'HelloWorldRecipe'});
   };
   //
+  const runArcSerialTest = () => {
+    // async `runArc` commands are performed serially
+    send({message: 'runArc', arcId: 'pipe-notification-test1', modality: 'dom', recipe: 'Notification'});
+    send({message: 'runArc', arcId: 'pipe-notification-test2', modality: 'dom', recipe: 'Notification'});
+    send({message: 'runArc', arcId: 'pipe-notification-test3', modality: 'dom', recipe: 'Notification'});
+  };
+  //
```

Comment:
sorry, could you clarify how this tests that these commands are indeed performed serially?

Reply:
It doesn't, you have to examine the console output. This is a 'testDevice' that produces smoke-output only, not a proper test.

Reply:
I just remembered that I previously added a wdio test that verify pipes-shell smoke test output as a form of testing, I'll have to figure out how to include this feature in that setup.